experimental/dds/tree2/src/test/feature-libraries/editable-tree/editableTree.editing.spec.ts

```diff
@@ -383,6 +383,113 @@ describe("editable-tree: editing", () => {
 		trees[0].context.free();
 	});
 
+	describe(`can move nodes`, () => {
+		it("to the left within the same field", () => {
+			const [provider, trees] = createSharedTrees(
+				getTestSchema(FieldKinds.sequence),
+				[{ type: rootSchemaName }],
+				2,
+			);
+			assert(isUnwrappedNode(trees[0].root));
+			assert(isUnwrappedNode(trees[1].root));
+			// create using `createFieldSymbol`
+			trees[0].root[createField](localFieldKey, [
+				singleTextCursor({ type: stringSchema.name, value: "foo" }),
+				singleTextCursor({ type: stringSchema.name, value: "bar" }),
+			]);
+			const field_0 = trees[0].root[localFieldKey];
+			assert(isEditableField(field_0));
+			assert.equal(field_0.length, 2);
+			assert.equal(field_0[0], "foo");
+			assert.equal(field_0[1], "bar");
+			assert.equal(field_0[2], undefined);
```

Comment:
editiableField is iterable so this should work:
```suggestion
			assert.deepEqual([...field_0], ["foo", "bar"]);
```

Using this pattern here and elsewhere will make this test a bit cleaner

Reply:
updated to simpler assert pattern

Reply:
you updated the ones below, but you missed this one.

Reply:
updated missed assert